src/views/UserProfile.vue

```diff
@@ -121,6 +126,50 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
                 </v-btn>
               </v-flex>
             </v-layout>
+
+            <v-layout row align-center wrap>
+              <v-flex xs3>
+                <span>Colour Theme</span>
+              </v-flex>
+                <v-radio-group v-model="jobTheme" column>
+                  <table class="c-job-state-table">
```

Comment:
Messed around with Vuetify components, however, a good old table seems to do the best job.